kimanetwork/chain.json

```diff
@@ -30,16 +30,40 @@
       {
         "denom": "uKIMA"
       }
-    ]
+    ],
+    "lock_duration": {
+      "time": "1814400s"
+    }
   },
   "codebase": {
     "git_repo": "https://github.com/kima-finance/kima-blockchain",
-    "recommended_version": "v0.4.3",
-    "compatible_versions": [
-      "v0.4.3"
-    ],
+    "recommended_version": "v0.4.5",
+    "compatible_versions": ["v0.4.5"],
     "genesis": {
       "genesis_url": "https://archive.kima.network/genesis.json"
+    },
+    "consensus": {
```

Comment:
The consensus, language, sdk, and ibc, are better stored in the "versions" file (but here is fine to include, as well).